.github/workflows/ci.yml

```diff
@@ -20,3 +20,16 @@ jobs:
           coverage: xdebug
       - run: composer install
       - run: vendor/bin/phpunit --coverage-text
+
+  static-analysis:
+    name: PHPStan
+    runs-on: ubuntu-20.04
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v3
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.1
+      - run: composer require phpstan/phpstan
+      - name: Execute type checking
```

Comment:
nit: All other steps don't use a `name`?

Reply:
Maybe they should all have a name, but this PR isn't the place to make that argument.